dashboards-observability/public/components/custom_panels/helpers/utils.tsx

```diff
@@ -47,7 +47,7 @@ export const convertDateTime = (datetime: string, isStart = true, formatted = tr
     returnTime = dateMath.parse(datetime, { roundUp: true });
   }
 
-  if (formatted) return returnTime.format(PPL_DATE_FORMAT);
+  if (formatted) return returnTime?.utc().format(PPL_DATE_FORMAT);
```

Comment:
`returnTime!.utc()`? `undefined?.utc().format` still throws exception

Reply:
Ah yes that makes sense.